src/applications/personalization/dashboard-2/components/Dashboard.jsx

```diff
@@ -121,20 +147,30 @@ const Dashboard = ({
 
               {showHealthCareError ? (
                 <div className="vads-l-row">
-                  <div className="vads-l-col--12 medium-screen:vads-l-col--8">
+                  <div className="vads-l-col--12 medium-screen:vads-l-col--8 medium-screen:vads-u-padding-right--3">
```

Comment:
Just a personal preference -- I would do 

```
showHealthCareError && (

) 
```

So you don't need the `null` - but it's fine either way 🙂

Reply:
I know, I think we talked about this sometime in the past month 😆  [It's just a little safer to use ternaries in JSX](https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx) so I'm trying to get into that habit.